jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1243,6 +1243,16 @@ void java_bytecode_parsert::rmethod_attribute(methodt &method)
   {
     rRuntimeAnnotation_attribute(method.annotations);
   }
+  else if(
+    attribute_name == "RuntimeInvisibleParameterAnnotations" ||
+    attribute_name == "RuntimeVisibleParameterAnnotations")
+  {
+    u1 parameter_count = read_u1();
+    if(method.parameter_annotations.size() < parameter_count)
```

Comment:
I believe resize would truncate the vector to the size required actually.

Reply:
Oops only read the bit in the box on the reference! That said shouldn't this vector be always empty here?

Reply:
To avoid the same question to any future reader, it would be worth to document the answer with either a comment or an assertion checking that the size is initially 0, followed by the `resize()`.